apps/drec-api/src/pods/device-group/device-group.entity.ts

```diff
@@ -16,5 +30,72 @@ export class DeviceGroup extends ExtendedBaseEntity implements IDeviceGroup {
   @Column()
   organizationId: number;
 
+  @Column()
+  @IsString()
+  fuelCode: string;
+
+  @Column()
+  @IsString()
+  countryCode: string;
+
+  @Column()
+  @IsEnum(StandardCompliance)
+  standardCompliance: StandardCompliance;
+
+  @Column('text', { array: true })
+  deviceTypeCodes: string[];
+
+  @Column({
+    type: 'enum',
+    enum: OffTaker,
+    array: true,
+    default: [],
+  })
+  offTakers: OffTaker[];
+
+  @Column({
+    type: 'enum',
+    enum: Installation,
+    array: true,
+    default: [],
+  })
+  installationConfigurations: Installation[];
+
+  @Column({
+    type: 'enum',
+    enum: Sector,
+    array: true,
+    default: [],
+  })
+  sectors: Sector[];
+
+  @Column({
+    type: 'enum',
+    enum: CommissioningDateRange,
+    array: true,
+    default: [],
+  })
+  commissioningDateRange: CommissioningDateRange[];
+
+  @Column()
+  @IsBoolean()
+  gridInterconnection: boolean;
+
+  @Column()
+  @IsNumber()
+  aggregatedCapacity: number;
+
+  @Column()
+  @IsEnum(CapacityRange)
+  capacityRange: CapacityRange;
+
+  @Column({ default: 1000 })
+  @IsNumber()
+  yieldValue: number;
+
+  @Column({ nullable: true })
+  @IsString()
+  labels: string;
```

Comment:
This does not reflect nullable: true

Reply:
You mean to add IsOptional() to it?

Reply:
No. If column can be null, then this field should be `labels: string | null`, right? Or `labels?: string` not sure

Reply:
Honestly it doesn`t really matter to postgress here. If labels is missing, it just adds null in the database. In the interface I defined it as labels?: string, but in the entity I don`t usually add the ?.